### PR TITLE
remove used tx outputs

### PR DESCRIPTION
**Pull request by igorcrevar**
remove used tx outputs

### CODE REVIEW

File: core/block_indexer.go [hunk 1]
```diff
@@ -301,3 +304,24 @@ func (bi *BlockIndexer) addAllTxOutputsToDb(dbTx DbTransactionWriter, txs []ledg
 		}
 	}
 }
+
+func (bi *BlockIndexer) getUsedUtxos(txs []*Tx) (res []*TxInput) {
```
Comment (babinskizoltan) [inline core/block_indexer.go:308]: maybe call func getTxInputs

File: core/block_indexer.go [hunk 2]
```diff
@@ -301,3 +304,24 @@ func (bi *BlockIndexer) addAllTxOutputsToDb(dbTx DbTransactionWriter, txs []ledg
...
+	for _, tx := range txs {
+		res = append(res, tx.Inputs...)
+	}
+
+	return res
+}
+
+func (bi *BlockIndexer) getAllUsedUtxos(txs []ledger.Transaction) (res []*TxInput) {
```
Comment (babinskizoltan) [inline core/block_indexer.go:316]: maybe call func getAllTxInputs

Comment (igorcrevar) [reply core/block_indexer.go]: fixed

**Review by sasa-ethernal — APPROVED** [review]
Looks good to me

**Comment by babinskizoltan** [comment]
Looks good

**Review by babinskizoltan — APPROVED** [review]

**Pull request merged**